include/tvSettings.h

```diff
@@ -2567,6 +2567,58 @@ tvError_t EnableDynamicContrast(bool mode);
  */
 tvError_t EnableLocalContrast(bool mode);
 
+/** 
+* @brief Triggers short circuit detection operation and returns the status.
+*
+* On calling GetLdimZoneShortCircuitStatus, the short circuit detection logic
+* is triggered. It detects short circuits between zones.  On completion, it
+* returns a list of zones where a short circuit was detected.
+*
+* @param[in/out] status
+*     - int*, Callee-saved status indicating short circuit detected or not.
+*     - 0 indicates success, no short circuit detected.
+*     - 1 indicates failure, at-least one short circuit detected.
+*     - This should be pre-allocated by the caller.
+*
+* @param[in/out] shortcircuit_zone_list
+*     - unsigned char*, each array element represents a zone short circuit status,
+*       like, [0][1][1][0]...[0] represents short detected or not.
+*       0 - no short circuit detected for the respective zone.
+*       1 - short circuit detected for the respective zone.
+*     - On input, this is an allocated array; on output, each indices
+*       represents zone number and it's value shows status of short
+*       detected/ not-detected status (on success).
+*     - This array should be pre-allocated by the caller with sufficient
+*       size to hold the maximum number of zones.
+*
+* @param[in] size
```

Comment:
What will be the max and min size?  I assume that it differs from platform to platform,?

Reply:
Yes, caller decides it based on platform

Reply:
Can we update the profile and provide different sizes that can be presented on platforms?  We should have some definitive size for each platform.

Reply:
Updated comments about the size param.  Pls check.